src/sctools/metrics/gatherer.py

```diff
@@ -230,3 +334,60 @@ def extract_metrics(self, mode: str = "rb") -> None:
                 # write a record for each gene id
                 metric_aggregator.finalize()
                 gene_metrics_output.write(gene_tag, vars(metric_aggregator))
+
+
+class GatherGeneMetricsFast(MetricGatherer):
```

Comment:
Same question as above (about a non-fast version that can be deprecated)

Reply:
non fast is deprecated now.

Reply:
if non-fast is deprecated, ti should be replaced or removed. We do npt need 2 separate paths through the code that produce the same output

Reply:
reverted back to original